.claude/scripts/python-ban-guard-go/main.go

```
@@ -0,0 +1,793 @@
+	path string
+	hits []string
+	seen map[string]bool
+}
+
+// main scans one supplied file and reports whether the compatibility route is needed.
+func main() {
+	if len(os.Args) != 3 {
+		fmt.Fprintln(os.Stderr, "python-ban-parser: need display path and input file")
+		os.Exit(2)
+	}
+	data, err := os.ReadFile(os.Args[2])
```

golangci-lint (2.13.2)

[high] G703: Path traversal via taint analysis

(gosec)